packages/stores/src/account/base.ts

```diff
@@ -948,10 +947,7 @@ export class AccountStore<Injects extends Record<string, any>[] = []> {
       ? wallet.client.signDirect(
           wallet.chainId,
           signerAddress,
-          {
-            ...signDoc,
-            accountNumber: Long.fromString(signDoc.accountNumber.toString()),
-          },
+          signDoc,
```

Comment:
Was there a guide for this? If you used a guide a link in the PR would be good for record keeping

If not, disregard

Reply:
No there isn't a guide I was just getting ts errors, I think this happens because now they use the latest version of cosmjs